siibra/volumes/providers/neuroglancer.py

```
@@ -205,35 +207,41 @@ def _merge_fragments(
-                voi=voi
             )
 
         num_conflicts = 0
         result = None
-        for frag_vol in self._fragments.values():
+        for frag_name, frag_vol in self._fragments.items():
             frag_scale = frag_vol._select_scale(
                 resolution_mm=resolution_mm,
                 bbox=voi,
                 max_bytes=kwargs.pop("maxbytes", NeuroglancerVolume.MAX_BYTES)
             )
             img = frag_scale.fetch(voi=voi)
+            if img is None:
+                logger.info(f"Fragment {frag_name} did not provide content for fetching.")
+                continue
             if result is None:
                 # build the empty result image with its own affine and voxel space
-                s0 = np.identity(4)
-                s0[:3, -1] = list(bbox.minpoint.transform(np.linalg.inv(img.affine)))
-                result_affine = np.dot(img.affine, s0)  # adjust global bounding box offset to get global affine
+                transl = np.identity(4)
+                transl[:3, -1] = list(bbox.minpoint.transform(np.linalg.inv(img.affine)))
+                result_affine = np.dot(img.affine, transl)  # adjust global bounding box offset to get global affine
                 voxdims = np.asanyarray(np.ceil(
                     bbox.transform(np.linalg.inv(result_affine)).shape  # transform to the voxel space
                 ), dtype="int")
                 result_arr = np.zeros(voxdims, dtype=img.dataobj.dtype)
                 result = nib.Nifti1Image(dataobj=result_arr, affine=result_affine)
 
             # resample to merge template and update it
-            resampled_img = resample_img_to_img(source_img=img, target_img=result)
+            try:
+                resampled_img = resample_img_to_img(source_img=img, target_img=result)
+            except BoundingBoxError:
+                logger.debug(f"Skipping fragment {frag_name} due to bounding box error")
+                continue
```

This might fail silently.

---

The debug message will be updated in the next commit